lib/bank_scrap.rb

```diff
@@ -7,6 +7,7 @@
 require 'bank_scrap/bank'
 require 'bank_scrap/account'
 require 'bank_scrap/transaction'
+require 'bank_scrap/banks/openbank/openbank'
```

Comment:
Is this really necessary? If it's possible, I would suggest using the same pattern as the other banks (with autoload).